packaging/packager_rpm.sh

```diff
@@ -5,40 +5,42 @@ readonly distro="${DISTRO:-fedora}"
 readonly distro_version="${VERSION:-30}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# rpm dependencies
-# python3
-dnf -y install python3 python3-coverage
-# python-imaging
+echo "------------- python3 -------------"
+dnf -y install python3 python3-coverage python3-devel
+echo "------------- python-imaging -------------"
 dnf -y install python3-pillow
-# pip dependencies (for dependencies not available as RPM)
+echo "-------------  pip dependencies (for dependencies not available as RPM) -------------"
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+echo "------------- upgrade pip  -------------"
 pip3 install --upgrade pip
-# contour, template, feature, cascade, text matching
+
+echo "------------- contour, template, feature, cascade and matching -------------"
```

Comment:
This should state the remaining types of matching when removing the text matching and we use Oxford comma, so "contour, template, feature, and cascade matching".